index.html

```diff
@@ -5,12 +5,42 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/4e655bdddb.js" crossorigin="anonymous"></script>
     <link rel="stylesheet" href="./style.css">
     <title>Portfolio</title>
 </head>
 
 <body>
-    <h1>Portfolio Project</h1>
+    <header>
+        <h1 class="logo"><a href="#">Usman</a></h1>
+        <nav>
+            <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <path fill-rule="evenodd" clip-rule="evenodd"
+                    d="M3 7C3 6.44772 3.44772 6 4 6H20C20.5523 6 21 6.44772 21 7C21 7.55228 20.5523 8 20 8H4C3.44772 8 3 7.55228 3 7ZM3 12C3 11.4477 3.44772 11 4 11L20 11C20.5523 11 21 11.4477 21 12C21 12.5523 20.5523 13 20 13L4 13C3.44772 13 3 12.5523 3 12ZM4 16C3.44772 16 3 16.4477 3 17C3 17.5523 3.44772 18 4 18H20C20.5523 18 21 17.5523 21 17C21 16.4477 20.5523 16 20 16H4Z"
+                    fill="#6070FF" />
+            </svg>
+        </nav>
+
+    </header>
```

Comment:
- [ ] KIndly make sure to add padding on both sides of the navigation bar. While you made changes on the navigation you have to make sure that your design still looks exactly as the provided Figma design. :wink: 

   - ### YOUR DESIGN
   
     ![image](https://user-images.githubusercontent.com/87197412/212103333-e3f94084-0910-4896-ba8f-806a97a05808.png)

   - ### FIGMA DESIGN
   
     ![image](https://user-images.githubusercontent.com/87197412/212103473-f90be0c8-aeb0-4774-bb59-8ce2d397b6c5.png)